app/models/session.js

```diff
@@ -3,7 +3,10 @@ module.exports = (sequelize, DataTypes) => {
     'session',
     {
       id: { type: DataTypes.INTEGER, allowNull: false, autoIncrement: true, primaryKey: true, unique: true },
-      userId: { allowNull: false, type: DataTypes.INTEGER, field: 'user_id', primaryKey: true }
+      userId: { allowNull: false, type: DataTypes.INTEGER, field: 'user_id', primaryKey: true },
+      paranoid: true,
+      timestamps: true,
+      deletedAt: 'destroyTime'
```

Comment:
Why did you declare this alias for `deletedAt`?

Reply:
Emmm actually, i just thought that was necessary